node/_crypto/crypto_browserify/browserify_rsa.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+// Copyright 2017 crypto-browserify. All rights reserved. MIT license.
+
+import { BN } from "./bn.js/bn.js";
+import { randomBytes } from "./randombytes.ts";
+
+function blind(priv) {
```

Comment:
I don't think this is proper blinding. `E(x) = (xr)e mod N` - It seems to be missing `mod N`

Reply:
ohh nvm, it's happening here https://github.com/denoland/deno_std/pull/1980/files#diff-e983dd7157f1536d25db1ae3aafa01d99cd6ba85c436dfcbb5afa5a0336bbeabR29